repose-aggregator/components/filters/keystone-v2-filter/src/main/scala/org/openrepose/filters/keystonev2/KeystoneV2Filter.scala

```diff
@@ -718,6 +720,9 @@ class KeystoneV2Filter @Inject()(configurationService: ConfigurationService,
     override def onLifecycleEvent(event: LifecycleEvents): Unit = {
       logger.debug(s"Received Lifecycle Event: $event")
     }
+
+    case class RegisteredFeed(id: String, unique: String)
```

Comment:
IntelliJ shenanigans. It likes inner classes to be declared at the bottom. It doesn't bother me at all, so I let it have its way.